dot/node.go

```diff
@@ -318,33 +318,41 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
 		return nil, err
 	}
 
-	telemetry.GetInstance().Initialise(!cfg.Global.NoTelemetry)
-
 	var telemetryEndpoints []*genesis.TelemetryEndpoint
 	if len(cfg.Global.TelemetryURLs) == 0 {
 		telemetryEndpoints = append(telemetryEndpoints, gd.TelemetryEndpoints...)
-
 	} else {
 		telemetryURLs := cfg.Global.TelemetryURLs
 		for i := range telemetryURLs {
 			telemetryEndpoints = append(telemetryEndpoints, &telemetryURLs[i])
 		}
 	}
 
-	telemetry.GetInstance().AddConnections(telemetryEndpoints)
-	genesisHash := stateSrvc.Block.GenesisHash()
-	err = telemetry.GetInstance().SendMessage(telemetry.NewSystemConnectedTM(
-		cfg.Core.GrandpaAuthority,
-		sysSrvc.ChainName(),
-		&genesisHash,
-		sysSrvc.SystemName(),
-		cfg.Global.Name,
-		networkSrvc.NetworkState().PeerID,
-		strconv.FormatInt(time.Now().UnixNano(), 10),
-		sysSrvc.SystemVersion()))
-	if err != nil {
-		logger.Debugf("problem sending system.connected telemetry message: %s", err)
+	if !cfg.Global.NoTelemetry {
+		telemetryLogger := logger.New(log.AddContext("pkg", "telemetry"))
```

Comment:
```suggestion
		telemetryLogger := logger.NewFromGlobal(log.AddContext("pkg", "telemetry"))
```

Otherwise it would race on os.Stdout (and/or interleave logs)